.travis.yml

```diff
@@ -35,6 +36,9 @@ cache:
 
 # logfile uploader credentials
 env:
+  matrix:
+    - COMMAND=check
+    - COMMAND=mango-on-travis
```

Comment:
This is going to explode our tests from currently 4 runs to 8 runs. We're already throttled with the number of simultaneous runs we can have, this is going to increase runtimes for Travis overall to at least double, if not worse.

I don't think we need the full crossbar approach. Wouldn't it be better to simply run `make check mango-on-travis` instead?